index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>My First Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+
+<body>
+<!-- first card box -->
+  <article class="firstcard">
+    <header class="top"><h1>Dull Playing Cards</h1></header>
+    <section>
+      <img src="images/cards1.jpg">
```

Comment:
This totally works, and is fine. Technically the requirements ask for a block level element that contains the image. (which is more of a challenge to do CSS wise) If you feel like revising that, it's decent practice.